src/scripting/game_lua_kernel.cpp

```diff
@@ -4266,6 +4264,21 @@ game_lua_kernel::game_lua_kernel(game_state & gs, play_controller & pc, reports
 	luaL_setfuncs(L, audio_callbacks, 0);
 	lua_setfield(L, -2, "audio");
 	lua_pop(L, 1);
+	
+	// Create the sync module
+	cmd_log_ << "Adding sync module...\n";
+	static luaL_Reg const sync_callbacks[] {
+		{ "invoke_command",    &intf_invoke_synced_command },
+		{ "run_unsynced",      &intf_do_unsynced },
+		{ "evaluate",          &intf_synchronize_choice },
+		{ "multi_evaluate",    &intf_synchronize_choices },
```

Comment:
@Pentarctagon Should it be `evaluate` and `evaluate_multiple` or should the first one be `evaluate_single`?

Reply:
`evaluate_multiple` and `evaluate_single` sounds good.